app/models/wiki_content/revision.rb

```diff
@@ -66,4 +67,10 @@ def plagiarism_report_link
     return unless ithenticate_id
     "/recent-activity/plagiarism/report?ithenticate_id=#{ithenticate_id}"
   end
+
+  def ref_tags_added
+    current_refs = self.features && self.features["feature.wikitext.revision.ref_tags"] || 0
```

Comment:
The guard here against `nil` features shouldn't be necessary, since that attribute is serialized to hash. If the value is `NULL` in the database, it will serialize as an empty hash.

`self.` is also not needed here. The Revision instance will be the default receiver of a method call, so we only need `self.` in cases where it overlaps with a Ruby keyword or similar edge cases (or when it's important for clarity).